tutorials/debug/overview_of_debugging_tools.rst

```diff
@@ -51,56 +53,109 @@ Sync Script Changes
 Any script that is saved will be reloaded on the running game.
 When used remotely on a device, this is more efficient with network filesystem.
 
+Script Editor Debug Tools and Options
```

Comment:
Note that we don't use American Title case for section titles in the Godot docs. This should be "Script editor debug tools and options".

Reply:
There's an exception for section titles that mirror property names in the engine which are capitalized, which is what is used in previous sections here, so I guess that's what lead you to use this style here.